src/super_gradients/training/datasets/samplers/all_samplers.py

```diff
@@ -2,10 +2,16 @@
 from super_gradients.training.datasets.samplers.infinite_sampler import InfiniteSampler
 from super_gradients.training.datasets.samplers.repeated_augmentation_sampler import RepeatAugSampler
 from torch.utils.data.distributed import DistributedSampler
-
+from torch.utils.data.sampler import Sampler, SequentialSampler, SubsetRandomSampler, RandomSampler, WeightedRandomSampler, BatchSampler
 
 SAMPLERS = {
     Samplers.INFINITE: InfiniteSampler,
     Samplers.REPEAT_AUG: RepeatAugSampler,
-    Samplers.DISTRIBUTED: DistributedSampler
+    Samplers.DISTRIBUTED: DistributedSampler,
+    Samplers.SAMPLER: Sampler,
+    Samplers.SEQUENTIAL: SequentialSampler,
+    Samplers.SUBSET_RANDOM: SubsetRandomSampler,
+    Samplers.RANDOM: RandomSampler,
+    Samplers.WEIGHTED_RANDOM: WeightedRandomSampler,
+    Samplers.BATCH: BatchSampler,
```

Comment:
Batch sampler is not a sampler...Its treated different and passed as a different argument to DataLoader constructor. See how we handle it inside datalaoders.get, and remove it from here please.

Reply:
removed